src/Command/CheckPasswordCommand.php

```diff
@@ -42,11 +48,13 @@ class CheckPasswordCommand extends ContainerAwareCommand
      * CheckPasswordCommand constructor.
      *
      * @param ObjectManager             $manager
+     * @param FileDescriptorReader      $reader
      * @param UserAuthenticationHandler $handler
      */
-    public function __construct(ObjectManager $manager, UserAuthenticationHandler $handler)
+    public function __construct(ObjectManager $manager, FileDescriptorReader $reader, UserAuthenticationHandler $handler)
```

Comment:
If the methods in FileDescriptorReader are static, we do not need to inject it in the contructor.

Reply:
If we don't inject `FileDescriptorReader`, then we cannot mock it in the unit tests, right? And that's the whole purpose of `FileDescriptorReader`, to mock it in unit tests in order to make `CheckPasswordCommand` testable. Or is there another way that I don't know of?